### PR TITLE
Add properties file support

### PR DESCRIPTION
**Pull request by Fatalll**

### CODE REVIEW

File: src/main/kotlin/tanvd/grazi/ide/language/props/PropsSupport.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package tanvd.grazi.ide.language.props
+
+import com.intellij.lang.properties.psi.impl.PropertyValueImpl
+import com.intellij.psi.PsiElement
+import tanvd.grazi.grammar.GrammarChecker
+import tanvd.grazi.grammar.Typo
+import tanvd.grazi.ide.language.LanguageSupport
+
+class PropsSupport : LanguageSupport() {
```
Comment (TanVD) [inline src/main/kotlin/tanvd/grazi/ide/language/props/PropsSupport.kt]: I would better rename it to `Properties`. At long last this is the name of plugin itself

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -104,6 +104,8 @@
 
         <languageSupport language="JSON" implementationClass="tanvd.grazi.ide.language.json.JsonSupport"/>
 
+        <languageSupport language="Properties" implementationClass="tanvd.grazi.ide.language.props.PropsSupport"/>
```
Comment (TanVD) [inline src/main/resources/META-INF/plugin.xml]: It looks like `Properties` plugin can be disabled. So we better move it to optional. (also move there JSON and XML)

File: src/test/resources/ide/language/props/Example.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+one.typo=It is <warning descr="ARTICLE_MISSING">friend</warning> of human
```
Comment (TanVD) [inline src/test/resources/ide/language/props/Example.properties]: :+1:

**Review by TanVD — CHANGES_REQUESTED** [review]

File: src/main/kotlin/tanvd/grazi/ide/language/props/PropsSupport.kt (hunk 1 above)
Comment (Fatalll) [reply src/main/kotlin/tanvd/grazi/ide/language/props/PropsSupport.kt]: Done

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (Fatalll) [reply src/main/resources/META-INF/plugin.xml]: Done

**Pull request merged**